src/components/card.js

```diff
@@ -17,8 +19,37 @@ const Card = (article) => {
   //   </div>
   // </div>
   //
+
+  const divCard = document.createElement('div');
+  const divHeadline = document.createElement('div');
+  const divAuthor = document.createElement('div');
+  const divImgCont = document.createElement('div');
+  const image = document.createElement('img');
+  const spanName = document.createElement('span');
+
+  divCard.classList.add('card');
+  divHeadline.classList.add('headline');
+  divAuthor.classList.add('author');
+  divImgCont.classList.add('img-container');
+
```

Comment:
If you wanted to get things out of an object without using dot notation you can do 
`let {headling, authorPhoto, authorName} = article `

Just another way to do what you did